### ISSUE TITLE
Some feeds stop updating after several days

### ISSUE THREAD

**Issue by maciejjo** [newsboat/newsboat#697]
**Newsboat version (copy the output of `newsboat -v` or the first line of `git show`)**:

```
newsboat 2.17.0 - https://newsboat.org/
Copyright (C) 2006-2015 Andreas Krennmair
Copyright (C) 2015-2019 Alexander Batischev
Copyright (C) 2006-2017 Newsbeuter contributors
Copyright (C) 2017-2019 Newsboat contributors

Newsboat is free software licensed under the MIT License. (Type `newsboat -vv' to see the full text.)
It bundles JSON for Modern C++ library, licensed under the MIT License: https://github.com/nlohmann/json

newsboat 2.17.0
System: Linux 4.19.75-1-lts (x86_64)
Compiler: g++ 9.1.0
ncurses: ncurses 6.1.20180127 (compiled with 6.1)
libcurl: libcurl/7.66.0 OpenSSL/1.1.1d zlib/1.2.11 libidn2/2.2.0 libpsl/0.21.0 (+libidn2/2.2.0) libssh2/1.8.2 nghttp2/1.39.2 (compiled with 7.66.0)
SQLite: 3.29.0 (compiled with 3.29.0)
libxml2: compiled with 2.9.9
```

Steps to reproduce the issue:

1. Add affected feed to URL list (one I know for sure is https://www.phoronix.com/rss.php)
2. After several days of updating, it will stop fetching new articles

I have no idea what is common factor here (which feeds are affected and which are not), I only observe this behavior with high volume feeds - because it is easy to notice that some feed that usually has many articles every day suddenly stops showing any. Refreshing the feed manually (by using 'r' when a feed is selected) completes seemingly without any error, it just does not add recent articles to the list. When I delete affected url from list, restart newsboat, re-add it to the list and start it again, it loads all recent items without any issue, and works correctly for another several days.

**Comment by Minoru** [comment]
Hi! Do you use `max-items` setting? If so, please update to 2.17.1 which fixes a relevant bug: #650

**Comment by maciejjo** [comment]
That was it! After update all feeds with missing items got updated. Thanks @Minoru!
